src/main/java/org/jabref/gui/preferences/PreviewTabView.java

```diff
@@ -197,10 +197,10 @@ private void jumpToSearchKey(ListView<PreviewLayout> list, KeyEvent keypressed)
             .findFirst().ifPresent(list::scrollTo);
     }
 
-    private void dragOver(DragEvent event) { ((PreviewTabViewModel) viewModel).dragOver(event); }
+    private void dragOver(DragEvent event) { viewModel.dragOver(event); }
 
     private void dragDetectedInAvailable(MouseEvent event) {
-        PreviewTabViewModel previewTabViewModel = (PreviewTabViewModel) viewModel;
+        PreviewTabViewModel previewTabViewModel = viewModel;
```

Comment:
Replace (inline) `previewTabViewModel` by `viewModel` ?

Reply:
huh, must have missed that one.